tests/components/fjaraskupan/conftest.py

```diff
@@ -0,0 +1,62 @@
+"""Standard fixtures for the Fjäråskupan integration."""
+from __future__ import annotations
+
+import asyncio
+from unittest.mock import patch
+
+from bleak.backends.device import BLEDevice
+from bleak.backends.scanner import AdvertisementData, BaseBleakScanner
+from pytest import fixture
+
+
+@fixture(name="scanner", autouse=True)
+def fixture_scanner(hass):
+    """Fixture for scanner."""
+
+    devices = [BLEDevice("1.1.1.1", "COOKERHOOD_FJAR")]
+
+    class MockScanner(BaseBleakScanner):
+        """Mock Scanner."""
+
+        async def start(self):
+            """Start scanning for devices."""
+            for device in devices:
+                self._callback(device, AdvertisementData())
+
+        async def stop(self):
+            """Stop scanning for devices."""
+
+        @property
+        def discovered_devices(self) -> list[BLEDevice]:
+            """Return discovered devices."""
+            return devices
+
+        def set_scanning_filter(self, **kwargs):
+            """Set the scanning filter."""
+
+    original_sleep = asyncio.sleep
+
+    async def sleeper(delay, result=None, *, loop=None):
+        await original_sleep(0)
+
+    with patch(
+        "homeassistant.components.fjaraskupan.config_flow.BleakScanner", new=MockScanner
+    ), patch(
+        "homeassistant.components.fjaraskupan.config_flow.asyncio.sleep", new=sleeper
```

Comment:
If possible it's better to patch the time slept. Make a constant for the time to sleep to make that patch easy.